dietpi/dietpi-software

```diff
@@ -11139,9 +11144,7 @@ bt-max-peers=$(Optimise_BitTorrent 2)
 listen-port=6881-6999
 check-certificate=false
 
-max-overall-upload-limit=0
 max-upload-limit=0
-max-overall-download-limit=0
 max-download-limit=0
```

Comment:
@msongz 
Why you removed the overall limits? They default to `0` anyway, but this is true for the per-torrent limits as well.

Reply:
😄